src/Promitor.Integrations.Azure/Authentication/Configuration/AuthenticationConfiguration.cs

```diff
@@ -4,5 +4,7 @@ public class AuthenticationConfiguration
     {
         public AuthenticationMode Mode { get; set; } = AuthenticationMode.ServicePrincipal;
         public string IdentityId { get; set; }
+        public string AppKeySecretFilePath { get; set; }
+        public string AppKeySecretFileName { get; set; }
```

Comment:
Can you elaborate on why you need this please?

Reply:
Also, why can we not just do `ApplicationKeyFilePath`?

Reply:
We use a sidecar container to download secrets from AKV and save it as a file, and mount this folder on the pod.
this is why we split folder name from file name, because we use the path the the mountPath attribute on the pod

Reply:
And that file contains a service principal secret then or do we need a new auth mode? We might want to split this in to a new PR maybe?

Reply:
Yes it is a support for a file that contains the secret instead of getting it from the configuration yaml, so it's not a new authentication mode

Reply:
Ok so let's leave this out and move this to a new PR that introduces a new authentication type.